src/rt-app_parse_config.c

```diff
@@ -830,14 +853,28 @@ parse_task_phase_data(struct json_object *obj,
 
 static void
 parse_task_data(char *name, struct json_object *obj, int index,
-		  thread_data_t *data, rtapp_options_t *opts)
+			thread_data_t *data, rtapp_options_t *opts)
```

Comment:
Indentation change? It looks broken in this view at least - but if it looks OK in a proper text editor then ignore me.

Reply:
I will check